image-sync/oc-mirror/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM registry.access.redhat.com/ubi9/ubi:9.4-1123.1719560047 AS downloader
+
+RUN dnf -y install unzip glibc-langpack-en
+
+ENV OC_VERSION=4.16.3
+
+RUN curl -sfL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux.tar.gz \
+    -o oc.tar.gz && \
+    tar -zvxf oc.tar.gz && \
+    mv oc kubectl /usr/local/bin
+
+RUN curl -sfL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/oc-mirror.tar.gz \
+    -o oc-mirror.tar.gz && \
+    tar -zvxf oc-mirror.tar.gz && \
+    mv oc-mirror /usr/local/bin
+
+FROM registry.access.redhat.com/ubi9/ubi:9.4-1123.1719560047
```

Comment:
the frontend dockerfile in this repo uses MSFT mariner base images. should we use them as well?

https://github.com/Azure/ARO-HCP/blob/main/frontend/Dockerfile

Reply:
I thought to change this to our mirrored image later. So, mirror this ubi image and build this container based on the ACR image.

Reply:
Going to switch it to: mcr.microsoft.com/cbl-mariner/distroless/base:2.0-nonroot@sha256:ef0dc582fc2a8dd34fbb41341a3a9a1aaa70d4542ff04ce4e33a641e52e4807e